m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -3008,6 +3029,9 @@ PROCEDURE PULLRGDATA
            NEWTRANSLDESCRIPTION=SHARE:NICKNAME
            IF NEWTRANSLDESCRIPTION="" THEN
             NEWTRANSLDESCRIPTION=SHARE:DESCRIPTION
+           CHARCHECK=NEWTRANSLDESCRIPTION
```

Comment:
Instead of doing the checking/scrubbing here, can we just do the check where we are outputting the json?